R/format_sl_summary.R

```diff
@@ -34,9 +34,9 @@
 #' meta <- meta_sl_example()
 #'
```

Comment:
Is it a good idea if we add a comment explaining what the table `mata |> prepare_sl_summary(...) |> formate_sl_summary(...)` is for? I guess it is a baseline character table. If so, is it equivalent to the one from `mata |> prepare_base_char(...) |> formate_base_char(...)`. If so, shall we say something here?